app/editor/src/features/content/form/utils/isSnippetForm.ts

```diff
@@ -16,3 +16,7 @@ export const isSnippetForm = (contentType: ContentTypeName) => {
       return true;
   }
 };
+
+export const isImageForm = (contentType: ContentTypeName) => {
```

Comment:
This file is for `isSnippetForm`.  If you place this function in this file it'll be hard to find it.  Make a separate file.

Reply:
Updated, thanks